parser/src/constraint.rs

```diff
@@ -158,7 +158,7 @@ impl Constraint {
         ensure!(!self.last_res.is_stop(), "compute_bias() called after stop");
         ensure!(
             self.step_arg.is_some(),
-            "commit_token() not called before compute_bias()"
+            "compute_bias() called twice without intervening commit_token()"
```

Comment:
Can you change this error message to reflect the (I assume new) function name "compute_mask"? There are a few occurrences of the name "compute_bias" before this too.

Reply:
Both a compute_mask() and a compute_bias() are present in the code, one calling the other.  You and @mmoskal should do whatever you think appropriate here.

Reply:
@hudson-ai If you haven't already, you might want to consult the chat for yesterday, where @mmoskal and I had a brief discussion of this message.  It was that discussion which motivated me to file this issue.

Reply:
@hudson-ai No problem.  This is a very easy call stack to get lost in!